src/components/tabBars.js

```diff
@@ -71,7 +71,7 @@ export const TabBar = ({
         href
       }, [
         div({
-          style: { flex: '1 1 100%', marginBottom: selected ? -(Style.tabBar.active.borderBottomWidth) : undefined }
+          style: { flex: '1 1 100%', height: 'inherit', marginBottom: selected ? -(Style.tabBar.active.borderBottomWidth) : undefined }
```

Comment:
This is the name of the "tab". It needs to stretch to the whole height of the navigation tab for the "active" indicator to render properly.